README.md

```diff
@@ -80,8 +80,8 @@ Parameters:
 	* `id`: a tile ID. provided in the fetch payload as `directoryId`; absent for history tiles
 	* `pin`: signals that the tile is pinned in the UA; absent for unpinned tiles
 	* `pos`: denotes the index of the tile in the new tab page; absent if the value is the same as its array index of `tiles`
-	* `score`: frecency for the tile; absent for frecencies under 5000
-	* `url`: signals that the tile is enhanced (with an empty "" value); absent for directory and history tiles
+	* `score`: frequency for the tile; absent for frequencies under 5000
```

Comment:
Actually, frecency is the right word here. It is determined according to this algorithm: https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Places/Frecency_algorithm